core/src/validator.rs

```diff
@@ -969,12 +969,30 @@ impl Validator {
                     )),
                     Some((&staked_nodes, &identity_keypair.pubkey())),
                 );
-                Arc::new(connection_cache)
+
+                let connection_cache_fwd = ConnectionCache::new_with_client_options(
+                    "connection_cache_tpu_fwd_quic",
+                    1,
+                    None,
```

Comment:
Can you plz add trailing comments what function arguments these values are for?
It saves a lot of time when someone reads the code. Like: 
```rust
1,    // connection_pool_size
None, // client_endpoint
```